views/beneficial-owner-type.html

```diff
@@ -0,0 +1,70 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  Who is registering - Register an overseas entity - Companies House
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
+
+  <div class="govuk-grid-row govuk-body">
+    <div class="govuk-grid-column-two-thirds">
+
+      <form method="post">
+
+        {% set ownerHint %}
+          {{ govukDetails({
+            summaryText: "What is a beneficial owner?",
+            text: "A beneficial owner is an individual who ultimately owns or controls more than 25% of a company's shares or voting rights, or who otherwise exercise control over the company or its management."
+          }) }}
+          <p class="govuk-hint">Select all that apply.</p>
+        {% endset %}
+
+        {{ govukCheckboxes({
+          idPrefix: "beneficial-owner-type",
+          name: "beneficial-owner-type",
+          fieldset: {
+            legend: {
+              text: "Who is a beneficial owner of the overseas entity?",
+              isPageHeading: true,
+              classes: "govuk-fieldset__legend--l"
+            }
+          },
+          hint: {
+            html: ownerHint
+          },
+          items: [
+            {
+              value: "individual",
+              text: "An individual beneficial owner"
+            },
+            {
+              value: "other-legal",
+              text: "A public authority, or other legal entity"
+            },
+            {
+              divider: "or"
+            },
+            {
+              value: "none",
+              text: "There is no beneficial owner",
+              behaviour: "exclusive"
+            }
+          ]
+        }) }}
+
+        {{ govukButton({
```

Comment:
May as well use the includes. Also no newline at end of file

Reply:
i'll change to use include